book/src/concepts/system.md

```diff
@@ -540,22 +540,3 @@ impl<'a> System<'a> for MyFirstSystem {
 }
 ```
 
-## The setup method
```

Comment:
I think it's worth keeping the info somehow, for the `new` method. It's difficult for new users to understand why we have the system data setup in `new` otherwise

Reply:
Still relevant.